cves/CVE-2015-1831.yml

```
@@ -207,4 +209,4 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: The problem was caused by poor design mistakes. The choice to have a the excludeParams function override those patterens in DefaultExcludedPatternsChecker was what caused the issue.
```

```suggestion
  answer: The problem was caused by poor design mistakes. The choice to have the excludeParams function override those patterns in DefaultExcludedPatternsChecker was what caused the issue.
```
"The problem was caused by poor design mistakes..." implies that there were multiple mistakes, but you only mention one of them, and state that it's **the cause** of the issue.

If you believe that there were multiple poor design decisions, you should either list the decisions, or specify that you're only talking about one of them:
  "For example, one of the mistakes was the choice to have the excludeParams function override..."

---

Description and grammar fixed.